src/legacy/core_plugins/embeddable_api/public/index.scss

```
@@ -1,13 +1,5 @@
 @import 'src/legacy/ui/public/styles/styling_constants';
 
-// Prefix all styles with "emb" to avoid conflicts.
-// Examples
-//   embChart
-//   embChart__legend
-//   embChart__legend--small
-//   embChart__legend-isLoading
-
-@import './variables';
-
-@import './panel/index';
-
+@import './css/variables';
```

The variables file should actually live right next to this `index` file
```suggestion
@import './variables';
```

---

In general, we try to stay away from `css`/`scss`/`styles` named folders as it promotes dumping of any CSS in there.

---

Done.